src/main/java/lotto/controller/Controller.java

```diff
@@ -0,0 +1,58 @@
+package lotto.controller;
+
+import lotto.domain.*;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Controller {
+
+    private static Controller controller = new Controller();
```

Comment:
```suggestion
    private static final Controller controller = new Controller();
```
최대한 불변을 유지해보시는게 어떨까요!?